Data/market-hours/market-hours-database.json

```diff
@@ -22758,6 +22758,260 @@
         "11/25/2022": "12:15:00"
       }
     },
+    "Future-cme-ETH": {
+      "dataTimeZone": "UTC",
+      "exchangeTimeZone": "US/Central",
```

Comment:
Just checking, but BTC is `America/Chicago`, micro eth & btc are `America/New_York` & ETH is `US/Central` (we have no other US/Central)?

Reply:
Right, updating with sources:
BTC: https://www.cmegroup.com/markets/cryptocurrencies/bitcoin/bitcoin.contractSpecs.html
Micro BTC (MBT): https://www.cmegroup.com/markets/cryptocurrencies/bitcoin/micro-bitcoin.contractSpecs.html
ETH: https://www.cmegroup.com/markets/cryptocurrencies/ether/ether.contractSpecs.html
Micro ETH (MET): https://www.cmegroup.com/markets/cryptocurrencies/ether/micro-ether.contractSpecs.html